packages/accounts-password/password_server.js

```diff
@@ -532,12 +532,12 @@ Accounts.generateVerificationToken = (userId, email, extraTokenData) => {
  * @returns {Object} Object with {email, user, token, url, options} values.
  * @importFromPackage accounts-base
  */
-Accounts.sendResetPasswordEmail = (userId, email, extraTokenData, extraParams) => {
+Accounts.sendResetPasswordEmail = async (userId, email, extraTokenData, extraParams) => {
   const {email: realEmail, user, token} =
     Accounts.generateResetToken(userId, email, 'resetPassword', extraTokenData);
   const url = Accounts.urls.resetPassword(token, extraParams);
   const options = Accounts.generateOptionsForEmail(realEmail, user, url, 'resetPassword');
```

Comment:
Can `Accounts.generateOptionsForEmail` also be made `async`, so that it can `await` each of the [user-provideable functions](https://github.com/meteor/meteor/blob/f60bfc7f75a24263539ef67fe856afab6eaf983e/packages/accounts-base/accounts_server.js#L1405-L1413) so that they too, may be async?